components/src/util/logger.js

```diff
@@ -0,0 +1,22 @@
+const isLogging = process.env.LOGGING;
+
+export default logger();
+
+function logger () {
+  return isLogging ? devLogger() : prodLogger();
+}
+
+function prodLogger () {
+  return {
+    log: noop,
+    info: noop,
+    error: noop,
+    warn: noop
+  };
+}
+
+function devLogger () {
+  return console;
```

Comment:
So right now I can do `logger.debug()` <- which will work in development but will be broken in production. Production and Development loggers should have the same APIs, otherwise you're production might be not working, but dev will.